### DISCUSSION TITLE
Send mails without a domain name

### DISCUSSION THREAD

**Discussion by RemyMachado** [postalserver/postal#2253] [Help with using Postal]
I'm trying to DNS setup the postal server on my VPS **without a domain name**.  
I successfully sent emails with the postal server ip address and now I'm trying to improve the configuration to avoid ending in spam folders.

My VPS provider told me I couldn't specify `SPF`, `DKIM` records without having a domain name. I'm confused.

___

### Is it possible to create these records without a domain name? 

My guess is that it's possible to create these records without a domain name, but it implies creating my own DNS.

**Comment by willpower232** [comment]
If you press the right buttons in Postal, you can probably get it to send whatever messages you want however email in general (and hence spam checking) tends to use other DNS records related to the domain name in the email address for context and other information so whilst you have apparently been able to send messages, it is unlikely that the receiving servers are capable of handling what you're doing and will probably relegate your messages to the spam folder.

tl;dr, it is really easier if you have a domain name 😅

**Reply by RemyMachado** [reply]
Hey, thanks for your prompt response.

I know it's easier with a domain name, every single tutorial I found was using one :confused:. I wanted to prove myself that it's possible, since I found people affirming that there is a way to make it work without a domain name.

Eventually, I gave up on this path, it's too much work and I don't have the required knowledge to lead this mission to success.

___

If someone knows a well documented way to **create the DNS records without a domain name** with `Postal`, please share it here.

**Reply by traderpedroso** [reply]
> Hey, thanks for your prompt response.
> 
> I know it's easier with a domain name, every single tutorial I found was using one 😕. I wanted to prove myself that it's possible, since I found people affirming that there is a way to make it work without a domain name.
> 
> Eventually, I gave up on this path, it's too much work and I don't have the required knowledge to lead this mission to success.
> 
> If someone knows a well documented way to **create the DNS records without a domain name** with `Postal`, please share it here.

First you don't need DNS records just fix IP if you want to send using IP it's simple user@ipaddress for sure you will get to spam or be blocked for example email like Gmail iCloud Hotmail  because can't set DKIM and SPF, but you can have 2 SMTP server using only fix IP and communication between 2
